config/v3_0_experimental/types/directory.go

```diff
@@ -29,7 +29,7 @@ func (d Directory) ValidateMode() report.Report {
 	}
 	if d.Mode == nil {
 		r.Add(report.Entry{
-			Message: errors.ErrPermissionsUnset.Error(),
+			Message: errors.ErrDirectoryPermissionsUnset.Error(),
```

Comment:
Super minor nit: We added the `AddOnWarning` recently function which could be used to make this less verbose. That can be deferred to a later PR though,

Reply:
:+1: thanks. I'll have a go at using this function now!